<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWidget</class>
 <widget class="QWidget" name="MainWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>874</width>
    <height>870</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>30</y>
     <width>801</width>
     <height>391</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="BoardView" name="graphicsView">
      <property name="styleSheet">
       <string notr="true">QGraphicsView{
border: none;
}</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="pSwitchLeft">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>430</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="CustomToolBar_1" name="tool_mid" native="true">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>430</y>
     <width>281</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="CustomToolBar_1" name="tool_left" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>430</y>
     <width>181</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="TabTools_1" name="tool_right" native="true">
   <property name="geometry">
    <rect>
     <x>610</x>
     <y>430</y>
     <width>231</width>
     <height>41</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="pSwitchRight">
   <property name="geometry">
    <rect>
     <x>570</x>
     <y>430</y>
     <width>31</width>
     <height>41</height>
    </rect>
   </property>
   <property name="sizePolicy">
    <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="PenPanel" name="penPanel" native="true">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>510</y>
     <width>151</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
  <widget class="EraserPanel" name="eraserPanel" native="true">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>510</y>
     <width>171</width>
     <height>111</height>
    </rect>
   </property>
  </widget>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CustomToolBar_1</class>
   <extends>QWidget</extends>
   <header>Custom/ToolBar/customtoolbar_1.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>TabTools_1</class>
   <extends>QWidget</extends>
   <header>Custom/TabTools/tabtools_1.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>BoardView</class>
   <extends>QGraphicsView</extends>
   <header>Draw/boardview.h</header>
  </customwidget>
  <customwidget>
   <class>PenPanel</class>
   <extends>QWidget</extends>
   <header>Panel/penpanel.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>EraserPanel</class>
   <extends>QWidget</extends>
   <header>Panel/eraserpanel.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
